UPDATING.md

```diff
@@ -31,6 +31,11 @@ some bugs.
 The new `sync_parallelism` config option will control how many processes CeleryExecutor will use to
 fetch celery task state in parallel. Default value is max(1, number of cores - 1)
 
+### New `log_processor_manager_location` config option
```

Comment:
I am not sure, but I thought the list is sorted by descending chronological order?

Reply:
Oh sry I didn't know that, if that is the case I'll update the order.